kolibri/deployment/default/settings/base.py

```diff
@@ -195,5 +195,10 @@
 
 AUTH_USER_MODEL = 'kolibriauth.DeviceOwner'
 
+AUTH_ANONYMOUS_USER_MODEL = "kolibri.auth.models.KolibriAnonymousUser"
```

Comment:
Quick word of caution: This is NOT a django setting, defining stuff here is absolutely 100% wrong.

Configuring stuff that's configurable elsewhere is okay.

It's like this:

yourapp.settings.your_variable

kolibri.deployment.default.settings.base = absolutely minimal settings skeleton which is NOT used for defining default settings of ANY kind.